### PR TITLE
(WIP) (Do not merge) Enable the Eco-Score API

### PR DESCRIPTION
**Pull request by teolemon**
This enables the alpha API

### CODE REVIEW

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/models/Product.java [hunk 1]
```diff
@@ -72,6 +72,8 @@ public String convert(String value) {
     private String customerService;
     @JsonProperty(ApiFields.Keys.ECOSCORE)
     private String ecoscore;
+    @JsonProperty(ApiFields.Keys.ECOSCORE_ALPHA_YES)
+    private String ecoscore_alpha_yes;
```
Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/models/Product.java]:
```suggestion
    @JsonProperty(ApiFields.Keys.ECOSCORE_ALPHA)
    private String ecoscore_alpha;
```

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/network/ApiFields.java [hunk 2]
```diff
@@ -90,6 +90,7 @@ private Defaults() {
         public static final String BRANDS = "brands";
         public static final String LC = "lc";
         public static final String ECOSCORE = "ecoscore_grade";
+        public static final String ECOSCORE_ALPHA_YES = "ecoscore_alpha_yes";
```
Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/network/ApiFields.java]:
```suggestion
        public static final String ECOSCORE_ALPHA = "ecoscore_alpha";
```

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -357,6 +357,7 @@
         <item>nutrition_grades_tags</item>
         <item>nutrient_levels</item>
         <item>ecoscore_grade</item>
+        <item>ecoscore_alpha_yes</item>
```
Comment (teolemon) [inline app/src/main/res/values/strings.xml]:
```suggestion
        <item>ecoscore_alpha</item>
```

**Comment by VaiTon** [comment]
What is this for?

**Comment by teolemon** [comment]
This is to enable the Eco-Score api for alpha testers.

**Comment by VaiTon** [comment]
@teolemon is this still needed?

**Comment by teolemon** [comment]
Nope we can close

**Pull request closed without merging**